apps/rewards/app/app-state-reducer.js

```diff
@@ -39,8 +41,9 @@ function appStateReducer(state) {
       reward.amountToken = amountToken.symbol
       return reward
     })
-    state.myRewards = state.myRewards || []
-    let metric = calculateAverageRewardsNumbers(state.rewards, state.claims, state.balances, updateConvertedRates(state.balances))
+    state.myRewards = state.rewards.filter(reward => reward.userRewardAmount > 0)
+    let convertRates = updateConvertedRates(state.balances)
+    let metric = calculateAverageRewardsNumbers(state.rewards, state.claims, state.balances, convertRates)
```

Comment:
Why `let` instead of `const`?

Reply:
Bad habit, this can be changed I'll switch it.